code/backend/batch/utilities/helpers/ConfigHelper.py

```diff
@@ -40,7 +43,7 @@ def __init__(self, config: dict):
         )
 
     def get_available_document_types(self):
-        return ["txt", "pdf", "url", "html", "md", "jpeg", "jpg", "png", "docx"]
+        return [processor.document_type for processor in self.document_processors]
```

Comment:
The problem with doing this is that once someone deletes a row from the Admin app, they won't be able to add that file type back

![image](https://github.com/Azure-Samples/chat-with-your-data-solution-accelerator/assets/60179183/2b9f7465-95e9-4596-8a55-f18cc52974e4)

Reply:
Urgh that sucks. I'll add the list back and add options if it's enabled.